Trying to deploy Strapi on Heroku with MySQL fails due to SSL settings.

**Describe the bug**
Trying to deploy Strapi on Heroku with MySQL fails due to SSL settings.

**Steps to reproduce the behavior**
1.- Open an account on Heroku.
2.- Create an app (following the Strapi-Heroku [deployment guide](https://strapi.io/documentation/3.0.0-beta.x/guides/deployment.html#heroku))
3.- Install the ClearDB MySQL plugin.
4.- Modify `/config/environments/production/database.json` to

```
{
  "defaultConnection": "default",
  "connections": {
    "default": {
      "connector": "strapi-hook-bookshelf",
      "settings": {
        "client": "mysql",
        "host": "${ process.env.DATABASE_HOST || '127.0.0.1' }",
        "port": "${ process.env.DATABASE_PORT || 3306 }",
        "srv": "${ process.env.DATABASE_SRV || false }",
        "database": "${ process.env.DATABASE_NAME || 'strapi' }",
        "username": "${ process.env.DATABASE_USERNAME || 'strapi' }",
        "password": "${ process.env.DATABASE_PASSWORD || 'strapi' }",
        "ssl": "${ process.env.DATABASE_SSL || true }"
      },
      "options": {
        "ssl": "${ process.env.DATABASE_SSL || true }",
        "authenticationDatabase": "${ process.env.DATABASE_AUTHENTICATION_DATABASE || '' }"
      }
    }
  }
}
```

**Expected behavior**
Deployment is OK and admin opens.

**System**

    Node.js version: v10.15.2
    NPM version: npm -v
    Strapi version: Strapi v3.0.0-beta.15
    Database: MySQL
    Operating system: Ubuntu 19.04

I'm having the same problem. Here is a printout of the error:

`> strapi start

[2019-11-02T16:48:13.934Z] error Impossible to register the 'category' model.
[2019-11-02T16:48:13.935Z] error TypeError: Unknown SSL profile ''
    at Function.getSSLProfile (/var/www/strapi/cms/node_modules/mysql/lib/ConneconConfig.js:34:24)
    at Object.createConnection (/var/www/strapi/cms/node_modules/mysql/index.js:13:34)
    at /var/www/strapi/cms/node_modules/knex/lib/dialects/mysql/index.js:65:38
    at Promise._execute (/var/www/strapi/cms/node_modules/bluebird/js/release/debuggability.js:411:9)
    at Promise._resolveFromExecutor (/var/www/strapi/cms/node_modules/bluebird/js/release/promise.js:518:18)
    at new Promise (/var/www/strapi/cms/node_modules/bluebird/js/release/promise.js:103:10)
    at Client_MySQL.acquireRawConnection (/var/www/strapi/cms/node_modules/knex/lib/dialects/mysql/index.js:64:12)
    at create (/var/www/strapi/cms/node_modules/knex/lib/client.js:287:39)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! cms@0.1.0 start: `strapi start`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the cms@0.1.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
`

--- a/packages/strapi-generate-new/lib/resources/json/database.json.js
+++ b/packages/strapi-generate-new/lib/resources/json/database.json.js
@@ -3,23 +3,17 @@
 module.exports = ({ connection, env }) => {
   // Production/Staging Template
   if (['production', 'staging'].includes(env)) {
-    // All available settings (bookshelf and mongoose)
     const settingsBase = {
       client: connection.settings.client,
       host: "${process.env.DATABASE_HOST || '127.0.0.1'}",
       port: '${process.env.DATABASE_PORT || 27017}',
-      srv: '${process.env.DATABASE_SRV || false}',
       database: "${process.env.DATABASE_NAME || 'strapi'}",
       username: "${process.env.DATABASE_USERNAME || ''}",
       password: "${process.env.DATABASE_PASSWORD || ''}",
-      ssl: '${process.env.DATABASE_SSL || false}',
     };
 
-    // All available options (bookshelf and mongoose)
     const optionsBase = {
-      ssl: '${process.env.DATABASE_SSL || false}',
-      authenticationDatabase:
-        "${process.env.DATABASE_AUTHENTICATION_DATABASE || ''}",
+      debug: false,
     };
 
     return {

--- a/packages/strapi-generate-new/lib/resources/json/database.json.js
+++ b/packages/strapi-generate-new/lib/resources/json/database.json.js
@@ -12,9 +12,7 @@ module.exports = ({ connection, env }) => {
       password: "${process.env.DATABASE_PASSWORD || ''}",
     };
 
-    const optionsBase = {
-      debug: false,
-    };
+    const optionsBase = {};
 
     return {
       defaultConnection: 'default',

